README.md

```diff
@@ -37,4 +39,49 @@ For a simplified, but less granular setup, you can use the API Token authenticat
 }
 ```
 
+An optional end_date field is can be added to the config.json 
```

Comment:
- "is can be added" ? :)
- can you use backquotes for parameters like `end_date`?
- same for `config.json`?